app/models/snack.rb

```diff
@@ -0,0 +1,9 @@
+class Snack < ApplicationRecord
+  has_many :machine_snacks
+  has_many :machines, through: :machine_snacks
+  validates_presence_of :name, :price
+
+  def display_price
+    "$#{sprintf('%.2f', price)}"
+  end
```

Comment:
same thing here, take a look at `number_to_currency`... but I really like how creative this is ;)